apps/heft/src/start.ts

```diff
@@ -3,6 +3,9 @@
 
 import { HeftToolsCommandLineParser } from './cli/HeftToolsCommandLineParser';
 
+// Load the Jest patch
+import './plugins/JestPlugin/jestWorkerPatch';
```

Comment:
Seems like we should only load this if we're actually going to be running tests. Can you just `require` it in the Jest plugin's execution before we load Jest?

Reply:
Heft doesn't currently delay load Jest, but I can move the import to that file so that the patch will be delay loaded if we implement that in the future.

The timing is important -- the patch needs to be applied before Jest starts doing stuff.

Reply:
Can we just delay-load Jest? Seems like that would improve boot times.

Reply:
Not in this PR